tests/kafkatest/version.py

```diff
@@ -69,7 +69,7 @@ def get_version(node=None):
         return DEV_BRANCH
 
 DEV_BRANCH = KafkaVersion("dev")
-DEV_VERSION = KafkaVersion("6.0.0-SNAPSHOT")
+DEV_VERSION = KafkaVersion("6.0.0-0")
```

Comment:
Have we tried running system tests against this PR? I remember we use some `LooseVersion` base class for this, but couldn't be sure off the top of my head if just changing this here and not making adjustments to the `KafkaVersion` implementation would work...

Reply:
@xli1996 ran the system tests and resolved the failures. The PR now contains those fixes.